packages/airnode-node/src/config/index.ts

```diff
@@ -25,6 +25,44 @@ const readConfig = (configPath: string): unknown => {
   return goParse.data;
 };
 
+export function addDefaultContractAddresses(config: configTypes.Config): configTypes.Config {
+  const ctx = { addIssue: () => {}, path: [] }; // Unused, but required by validator functions
+  const chains = config.chains.map((chain) => {
+    const contracts = configTypes.ensureValidAirnodeRrp(chain.contracts, chain.id, ctx);
+
+    const crossChainRequesterAuthorizers = chain.authorizers.crossChainRequesterAuthorizers
+      ? chain.authorizers.crossChainRequesterAuthorizers.map((craObj) => {
+          return configTypes.ensureCrossChainRequesterAuthorizerValidAirnodeRrp(craObj, ctx);
+        })
+      : [];
+
+    const crossChainRequesterAuthorizersWithErc721 = chain.authorizers.crossChainRequesterAuthorizersWithErc721
+      ? chain.authorizers.crossChainRequesterAuthorizersWithErc721.map((craObj) => {
+          return configTypes.ensureCrossChainRequesterAuthorizerWithErc721(craObj, ctx);
+        })
+      : [];
```

Comment:
Are the conditions necessary? The `chain.authorizers.crossChainRequesterAuthorizers` and `chain.authorizers.crossChainRequesterAuthorizersWithErc721` must be present but they can be empty arrays. So, mapping through them even if they are empty will lead to the same thing, an empty array.

Reply:
Good call 👍